### DISCUSSION TITLE
Is there a way to increase font size for the sims 1? It is impossible to read on 2k resolution.

### DISCUSSION THREAD

**Discussion by glorsh66** [FaithBeam/Sims-1-Complete-Collection-Widescreen-Patcher#67] [General]
Is there a way to increase font size for the sims 1? It is impossible to read on 2k resolution.

**Comment by FaithBeam** [comment]
I haven't been able to figure out increasing zoom or increasing font size, unfortunately. I stick to 1280x720 or 1366x768 resolutions to get the game widescreen but still legible.
